app/lib/services/voip/voipDebugLogger.ts

```
@@ -0,0 +1,68 @@
+			} catch (e) {
+				lastError = String(e);
+			}
+		})
+		.catch(() => {
+			// never break the chain
+		});
+};
+
+export const voipDebugLog = (tag: string, msg: string, data?: unknown) => {
+	const line = `${ts()} [${tag}] ${msg}${data !== undefined ? ` ${safeStringify(data)}` : ''}`;
+	(globalThis as any).console['log'](line);
```

ESLint

[error] 'globalThis' is not defined.

(no-undef)